### PR TITLE
Photo schema

### PR DESCRIPTION
**Pull request by marcellino-ornelas**

### CODE REVIEW

File: models/Photo.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const PhotoSchema = new Schema({
+  room_id: { type: Number, required: true },
+  photo_url: { type: String },
+  description: { type: String },
+  verified: { type: Boolean }
+});
+
+
+var Photo = mongoose.model( 'Photo', PhotoSchema );
```
Comment (rroque98) [inline models/Photo.js]: change to const

File: generate.js [hunk 2]
```diff
@@ -1 +1,2 @@
-var db = require('./models/');
\ No newline at end of file
+var db = require('./models');
```
Comment (rroque98) [inline generate.js:1]: Consider changing var to const

File: generate.js [hunk 3]
```diff
@@ -1 +1,2 @@
...
+var faker = require('faker');
```
Comment (rroque98) [inline generate.js:2]: Change var to const

**Review by rroque98 — CHANGES_REQUESTED** [review]

File: generate.js (hunk 3 above)
Comment (marcellino-ornelas) [reply generate.js:2]: Fixed `const` variable declarations. Dont worry about `var` on generate.js. I fix this problem in another branch

Comment (rroque98) [reply generate.js:2]: Ok, sounds good

**Review by rroque98 — APPROVED** [review]

**Pull request merged**